app/src/Page/Menu.elm

```diff
@@ -12,76 +17,291 @@ import Bulma
+    { retroList = []
     , retroName = ""
-    } ! [ getRetros ]
+    , possibleParticipants = []
+    , participants = []
+    , participant = ""
+    , autocompleteState = Autocomplete.empty
+    , currentChoice = Nothing
+    }
 
-type Msg = GotRetros (Result Http.Error (List String))
-         | CreateRetro
-         | SetRetroName String
+mount : Sock.Sender Msg -> Cmd Msg
+mount sender =
+    Sock.menu sender
 
-getRetros : Cmd Msg
-getRetros =
-     Http.get "/retros" (Decode.list Decode.string)
-         |> Http.send GotRetros
+subscriptions : Model -> Sub Msg
+subscriptions model =
+    Sub.map SetAutoState Autocomplete.subscription
 
-createRetro : String -> Cmd Msg
-createRetro name =
-    Http.post "/retros" (Http.jsonBody (Encode.string name)) (Decode.list Decode.string)
-        |> Http.send GotRetros
 
-update : Msg -> Model -> (Model, Cmd Msg)
-update msg model =
+type Msg = CreateRetro
+         | SetRetroName String
+         | AddParticipant
+         | SetParticipant String
+         | DeleteParticipant String
+         | SetAutoState Autocomplete.Msg
+         | SelectParticipant String
+         | ShowRetroDetails String
+
+
+update : Sock.Sender Msg -> Msg -> Model -> (Model, Cmd Msg)
+update sender msg model =
     case msg of
         SetRetroName input ->
             { model | retroName = input } ! []
 
         CreateRetro ->
-            model ! [ createRetro model.retroName ]
+            model ! [ Sock.createRetro sender model.retroName model.participants ]
+
+        SetParticipant input ->
+            { model | participant = input } ! []
+
+        AddParticipant ->
+            { model
+                | participant = ""
+                , participants = model.participant :: model.participants }
+            ! []
 
-        GotRetros resp ->
-            case resp of
-                Ok retros ->
-                    { model | retroList = Just retros } ! []
+        DeleteParticipant name ->
+            { model | participants = List.filter ((/=) name) model.participants } ! []
 
-                _ ->
-                    model ! []
+        SelectParticipant name ->
+            { model | participants = name :: model.participants } ! []
+
+        ShowRetroDetails retroId ->
+            { model | currentChoice = List.head <| List.filter (\x -> x.id == retroId) model.retroList } ! []
+
+        SetAutoState autoMsg ->
+            let
+                (newState, maybeMsg) =
+                    Autocomplete.update
+                        updateConfig
+                        autoMsg
+                        5
+                        model.autocompleteState
+                        (acceptablePeople "" model)
+
+                newModel =
+                    { model | autocompleteState = newState }
+            in
+                case maybeMsg of
+                    Just newMsg ->
+                        update sender newMsg newModel
+                    Nothing ->
+                        newModel ! []
```

Comment:
I'm not really a fan of this package, maybe just write it myself since it shouldn't be too complex. All I need is something that shows a list of options when you type and then adds the one you click.